.lintrunner.toml

```diff
@@ -463,6 +463,7 @@ include_patterns = [
 ]
 exclude_patterns = [
     'torch/csrc/utils/pybind.h',
+    'torch/csrc/dynamo/**/*',
```

Comment:
Why?  Can we just fix the lints?

Reply:
I was experimenting to see if including `torch/csrc/utils/pybind.h` over `pybind/pybind11.h` was causing issues in CI. I don't think this is the case - I've reverted the lintrunner settings.